composer.json

```diff
@@ -29,6 +29,19 @@
             "type": "composer",
             "url": "https://asset-packagist.org"
         },
+        {
+            "type": "package",
```

Comment:
I do not see any reason why this library should be added.

Reply:
The change aims to determine the local time of the YMCA branch location purely in JavaScript in the end user's browser. To do this we need to have complete timezone and daylight savings information in JavaScript. This is what the Luxon library provides.